visual-js/visual-cypress/src/index.ts

```diff
@@ -320,104 +298,89 @@ Sauce Labs Visual: Unable to create new build.
         retry: (error?: Error) => {
           return !!error && error instanceof DiffNotReadyError;
         },
-        // Will wait for 800ms + jitter with current settings
         // See https://www.npmjs.com/package/exponential-backoff#backoffoptions for details
-        delayFirstAttempt: false,
+        delayFirstAttempt: true,
```

Comment:
Updated these values to provide a better experience for delays. Previous amount was way too short for waiting on async processing.